File: KingdomCodeTool/wwwroot/Download/202503050759/Angular/shared/DanhMucMayChamCong.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucMayChamCong } from './DanhMucMayChamCong.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucMayChamCongService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'ID', 'ParentID', 'ParentName', 'CreatedDate', 'CreatedMembershipID', 'LastUpdatedDate', 'LastUpdatedMembershipID', 'RowVersion', 'SortOrder', 'Active', 'TypeName', 'Name', 'Code', 'Note', 'Display', 'FileName', 'Description', 'HTMLContent', 'DanhMucNgonNguID', 'DanhMucUngDungID', 'DanhMucNgonNguName', 'DanhMucUngDungName'];

    List: DanhMucMayChamCong[] | undefined;
    ListFilter: DanhMucMayChamCong[] | undefined;
    FormData!: DanhMucMayChamCong;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucMayChamCong";
    }
}

File: KingdomCodeTool/wwwroot/Download/202503050759/Service/Implement/DanhMucMayChamCongService.cs
namespace Service.Implement
{
    public class DanhMucMayChamCongService : BaseService<DanhMucMayChamCong, IDanhMucMayChamCongRepository>
    , IDanhMucMayChamCongService
    {
    private readonly IDanhMucMayChamCongRepository _DanhMucMayChamCongRepository;
    public DanhMucMayChamCongService(IDanhMucMayChamCongRepository DanhMucMayChamCongRepository) : base(DanhMucMayChamCongRepository)
    {
    _DanhMucMayChamCongRepository = DanhMucMayChamCongRepository;
    }
    }
    }

